metaflow/cli.py

```diff
@@ -38,6 +41,12 @@
 LOGGER_COLOR = 'green'
 LOGGER_BAD_COLOR = 'red'
 
+# Defaults for running MF on KFP
+DEFAULT_FLOW_CODE_URL = 'https://gist.githubusercontent.com/mon95/98d9d88571a0307a53b637d841295702/raw/df8bc1fa363f629271cfc1d34f3c2f7ca0b2e2cf/helloworld.py'
```

Comment:
please document a todo for why `DEFAULT_FLOW_CODE_URL` exists.

Reply:
done